Makefile

```diff
@@ -9,7 +9,7 @@ ifeq ($(RUST_BACKEND),ironox)
 	CARGO := $(RUST_ROOT)/build/x86_64-unknown-linux-gnu/stage0/bin/cargo
 else ifeq ($(RUST_BACKEND),cranelift)
 	CLIF_SO := librustc_codegen_cranelift.so
-	BACKEND := $(realpath $(CLIF_DIR)/target/debug/$(CLIF_SO))
```

Comment:
Thank you for spotting this!

Reply:
It was actually correct before I changed `test.sh` to `./test.sh --release`.

Reply:
Yes, but I should've been running Cranelift in release mode.